### ISSUE TITLE
github action for datalad-osf

### ISSUE THREAD

**Issue by hvgazula** [datalad/datalad-osf#193]
**Is your feature request related to a problem? Please describe.**
I just found out there's a github action available for installing datalad. However, my storage siblings are on OSF meaning, I need to install datalad-osf as well. Currently, I do this by `pip install datalad-osf`. 

**Proposed solution & alternatives considered**
While I am not an expert in github actions, I was curious if there are any thoughts of creating an action for datalad-osf as well.

**Comment by yarikoptic** [comment]
do you mean like adding `pip install datalad-osf` to accompany installation of `datalad` in `datalad-action` before it tries to fetch any data from OSF? then we better do it in https://github.com/datalad/datalad-action , right?

**Comment by hvgazula** [comment]
Yes, by providing it as an option with a True or False flag along with datalad installation (as not every user may need the osf part).

**Comment by yarikoptic** [comment]
ok, let's move there and I am looking into addressing it but rather a little differently  (to gain any generic `pip install`) in https://github.com/datalad/datalad-action/pull/17
